src/mimeparser.rs

```diff
@@ -1936,6 +1937,8 @@ fn get_mime_type(
         },
         mime::AUDIO => Viewtype::Audio,
         mime::VIDEO => Viewtype::Video,
+        mime::VCARD => Viewtype::Vcard,
+        // TODO(vcard): Handle TEXT_VCARD?
```

Comment:
Not sure what is to do here?

Reply:
I just didn't get why there are `mime::VCARD` and `mime::TEXT_VCARD`, probably we need to handle the latter because we send vCards as `text/vcard`, see `guess_msgtype_from_suffix()`

Reply:
Fixed in 2454eaaff, we're matching on `mimetype.type_()` which is `text` in the case of vcards, we need to match on the subtype which is `vcard` in the case of vcards. `mime::TEXT_VCARD` is equal to the string `text/vcard`, which we don't need here.